docs/known-issues.md

```diff
@@ -1,14 +1,11 @@
-| Running a worker that has been built using IL2CPP leads to a crash. This is due to [a bug in Unity](https://issuetracker.unity3d.com/issues/il2cpp-notsupportedexception-is-thrown-when-using-system-dot-type-dot-getcustomattributes-type-bool).                                                                                                         | 2019/01/28 | None                                                                                                | Use the Mono compiler. There is no workaround for iOS.                                                |
-| Running the FPS project in the Editor with the current platform as Android leads to the walls being slightly transparent.                                                                                                      | 2019/01/28 | None                                                                                                | It renders correctly when running in the Emulator or an Android device                                                |
-| Building a worker in the FPS starter project for Linux will throw errors when compiling the 'Hidden/PostProcessing/FinalPass' shader.                                                                                                    | 2019/01/28 | None                                                                                                | The build still succeeds and won't affect your worker.                                                |
-| Protocol logging on Linux workers is disabled due to a libcpp linking bug in Unity.                                                                                                   | 2019/01/28 | None                                                                                                | No workaround.                                                |
+| Running the FPS Starter Project in the Unity Editor with the current platform as Android leads to the walls being slightly transparent.                                                                                                      | 2019/01/28 | None                                                                                                | It renders correctly when running in the Emulator or an Android device                                                |
+| Building a worker in the FPS Starter Project for Linux will throw errors when compiling the `Hidden/PostProcessing/FinalPass` shader.                                                                                                    | 2019/01/28 | None                                                                                                | The build still succeeds and won't affect your worker.                                                |
```

Comment:
```suggestion
| Building a worker in the FPS Starter Project for Linux throw errors when compiling the `Hidden/PostProcessing/FinalPass` shader.                                                                                                    | 2019/01/28 | None                                                                                                | The build still succeeds and won't affect your worker.                                                |
```

Reply:
If we get rid of "will" then we need to make "throw" into "throws" so accepting the change and then re-editing. And I don't like "will" either.